uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt5/client/paho/MqttConnectionImpl.java

```diff
@@ -417,63 +420,7 @@ private class MqttMessageListener implements IMqttMessageListener {
-            Integer messageExpiryInterval = null;
-            String responseTopic = null;
-            byte[] correlationData = null;
-            if (receivedProperties != null) {
-                contentType = receivedProperties.getContentType();
-                payloadFormatIndicator = receivedProperties.getPayloadFormat();
-                if (receivedProperties.getMessageExpiryInterval() != null) {
-                    messageExpiryInterval = receivedProperties.getMessageExpiryInterval().intValue();
-                }
-                responseTopic = receivedProperties.getResponseTopic();
-                correlationData = receivedProperties.getCorrelationData();
-            }
-
-            List<Mqtt5Properties> userProps = convertToMqtt5Properties(receivedProperties);
-
-            if (isClosing.get()) {
-                logger.atWarn().log("PIBLISH event ignored due to shutdown initiated");
-            } else {
-                GRPCClient.MqttReceivedMessage message = new GRPCClient.MqttReceivedMessage(
-                        mqttMessage.getQos(), mqttMessage.isRetained(), topic,
-                        mqttMessage.getPayload(), userProps, contentType, payloadFormatIndicator,
-                        messageExpiryInterval, responseTopic, correlationData);
-                executorService.submit(() -> {
-                    try {
-                        grpcClient.onReceiveMqttMessage(connectionId, message);
-                    } catch (Exception ex) {
-                        logger.atError().withThrowable(ex).log("onReceiveMqttMessage failed");
-                    }
-                });
-            }
-
-            logger.atInfo().log("Received MQTT message: connectionId {} topic '{}' QoS {} retain {}",
-                            connectionId, topic, mqttMessage.getQos(), mqttMessage.isRetained());
-
-            if (userProps != null) {
-                userProps.forEach(p -> logger.atInfo()
-                        .log("Received MQTT userProperties: {}, {}", p.getKey(), p.getValue()));
-            }
-            if (contentType != null) {
-                logger.atInfo().log("Received MQTT message has content type '{}'", contentType);
-            }
-            if (payloadFormatIndicator != null) {
-                logger.atInfo().log("Received MQTT message has payload format indicator '{}'", payloadFormatIndicator);
-            }
-            if (messageExpiryInterval != null) {
-                logger.atInfo().log("Received MQTT message has message expiry interval {}", messageExpiryInterval);
-            }
-            if (responseTopic != null) {
-                logger.atInfo().log("Received MQTT message has response topic: {}", responseTopic);
-            }
-            if (correlationData != null) {
-                logger.atInfo().log("Received MQTT message has correlation data: {}", correlationData);
-            }
+            MqttConnectionImpl.this.messageArrived(topic, mqttMessage);
```

Comment:
Remove MqttConnectionImpl.this. when private method will be renamed.

Reply:
fixed in dfd2cb402305dcd47a03070e302a2a1f0592d8f7